Source/WebCore/workers/WorkerConsoleClient.cpp

```diff
@@ -89,13 +89,33 @@ void WorkerConsoleClient::timeEnd(JSC::JSGlobalObject* exec, const String& label
         InspectorInstrumentation::stopConsoleTiming(*worker, exec, label);
 }
 
-// FIXME: <https://webkit.org/b/153499> Web Inspector: console.profile should use the new Sampling Profiler
-void WorkerConsoleClient::profile(JSC::JSGlobalObject*, const String&) { }
-void WorkerConsoleClient::profileEnd(JSC::JSGlobalObject*, const String&) { }
+void WorkerConsoleClient::profile(JSC::JSGlobalObject*, const String& title)
+{
+    // FIXME: Add support for WorkletGlobalScope.
```

Comment:
I'm not exactly sure of the difference between Worklet and Worker here. Should this just be fixed as part of this change, or does it need a followup bug?

Reply:
worklets are a more lightweight version of a `Worker` that serve a specific purpose (e.g. `AudioWorklet`)

i dont think we have any Web Inspector support for them right now, but eventually it's something we'd likely want

Reply:
Okay. Let's get a bugzilla link and put it into these worklet FIXMEs.